src/main/kotlin/org/rust/cargo/toolchain/tools/Cargo.kt

```diff
@@ -175,14 +175,15 @@ class Cargo(
     fun fetchMetadata(
         owner: Project,
         projectDirectory: Path,
-        buildTarget: String?,
+        buildTargets: List<String>,
         toolchainOverride: String? = null,
         listener: ProcessListener? = null
     ): RsResult<CargoMetadata.Project, RsProcessExecutionOrDeserializationException> {
         val additionalArgs = mutableListOf("--verbose", "--format-version", "1", "--all-features")
-        if (buildTarget != null) {
+        buildTargets.forEach {
```

Comment:
```suggestion
        for (it in buildTargets) {
```

I'd prefer using for over `forEach`